Makefile

```diff
@@ -165,6 +165,16 @@ push: image
 	docker push ${REGISTRY}/yunikorn:scheduler-${VERSION}
 	docker push ${REGISTRY}/yunikorn:admission-${VERSION}
 
+#Generate the CRD code with code-generator (release-1.14)
+
+# If you want to re-run the code-generator to generate code,
+# Please make sure the directory structure must be the example.
+# ex: github.com/apache/incubator-yunikorn-k8shim
```

Comment:
Is it possible to recognise where the repository is checked out instead of using a hardcoded directory structure? I tried to generate the code and in order to have it generated I have to change all my directory structure, what is a bit painfull. 
It will also fail, if GOPATH is not set. It would be good to set it in the script, or at least document it that we need it set. For me a simple " export GOPATH="$HOME/go"  solved the issue.

Reply:
If we change the structure of directory, it may affect the import path which in the generated file.
About the GOPATH, if GOPATH is empty, it will export GOPATH in generate-groups.sh, which set in sub-shell, won't bring back to parent shell.The GOPATH in environment should be set by user.

Reply:
Thank you for the explanation. It makes sense